src/objects.rs

```diff
@@ -674,8 +674,65 @@ impl Coord {
 
         2. * EARTH_RADIUS * f64::asin(f64::sqrt(x + y))
     }
+
+    /// Returns a proxy object allowing to compute approximate
+    /// distances for cheap computation.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use navitia_model::objects::Coord;
+    /// let v: Vec<Coord> = vec![];
```

Comment:
why v is empty in your example?

Reply:
because I'm lazy off adding a lot of coords, and that's not so useful for the example. I'll mask that with a function.